aws_jupyter_proxy/awsproxy.py

```diff
@@ -158,6 +161,16 @@ def __init__(
         and some operations send such requests (such as S3.InitiateMultipartUpload)
         :return: the HTTPResponse
         """
+        if self.whitelisted_services is not None and self.service_info.service_name not in self.whitelisted_services.strip(
```

Comment:
Do this splitting etc once in the `__init__` so that this check becomes a single hashmap lookup

Reply:
I like it for readability, but for performance will be almost the same because a new ‘AwsRequestProxy’ object is instantiated per request

Reply:
Can you set the variable globally? I think it's less readable this way - in the request we don't care about parsing environment variables we only care if the service is whitelisted or not.

Reply:
> I think it's less readable this way

I meant your suggestion of doing it in `__init__` is more readable.

Reply:
Woops haha :)